docs/platforms/javascript/common/configuration/filtering.mdx

```diff
@@ -220,6 +220,5 @@ Learn more about <PlatformLink to="/configuration/sampling/">configuring the sam
 ## Filtering Spans
 
 To prevent certain spans from being reported to Sentry, use the <PlatformIdentifier name="before-send-span" /> configuration option, which allows you to provide a function to evaluate the current span and drop it if it's not one you want.
-This API is available from Cocoa SDK version 8.30.0 and above.
```

Comment:
we could change it to SDK version 8.2.0 -> https://github.com/getsentry/sentry-javascript/blob/develop/CHANGELOG.md#820